longest_palindrome.cpp

```diff
@@ -0,0 +1,23 @@
+class Solution {
```

Comment:
Time and space here?

Reply:
Space is going to be constant cause maximum number of elements in the hashset will be the total number of alphabets. Time complexity will be O(string size) because we are going to traverse through the string. Please correct me if I'm wrong. @super30admin

Reply:
Time?